kayenta-core/src/main/java/com/netflix/kayenta/canary/ExecutionMapper.java

```diff
@@ -52,6 +52,7 @@
 @Slf4j
 public class ExecutionMapper {
 
+  public static final String PIPELINE_NAME = "Standard Canary Pipeline";
   private final StorageServiceRepository storageServiceRepository;
```

Comment:
Could add a blank line between the publics and privates.

Reply:
- [X] done